Mapping/Loader/AnnotationsLoader.php

```diff
@@ -190,6 +189,13 @@ public function parseControllerMappings(
         $reflectionMethod = new \ReflectionMethod($controller, $method);
         $methodAnnotations = $this->annotationReader->getAnnotations($reflectionMethod);
 
+        if (\PHP_VERSION_ID >= 80000 && count($attributes = $reflectionMethod->getAttributes(SubscribeTo::class, \ReflectionAttribute::IS_INSTANCEOF)) > 0) {
+            foreach ($attributes as $attribute) {
+                $annotation = $attribute->newInstance();
+                $this->parseSubscribeTo($annotation, $routeName, $route, $subscriptions);
+            }
+        }
+
         foreach ($methodAnnotations as $class => $annotations) {
```

Comment:
jel bi ovaj foreach trebao ici pod else mozda? da se ne zove `$this->parseSubsrcibeTo `duplo u slucaju da je php verzija > 8

Reply:
@shule727 Moja ideja je da se hendlaju i anotacije i atributi, tj da mogu paralelno postojati i jedni i drugi na istoj metodi. Malo sam statemente reorderao i dodao komentar da bude malo jasnije.